Assets/Scripts/WallLightController.cs

```diff
@@ -0,0 +1,147 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class WallLightController : MonoBehaviour {
+    private const string REPLACE_BULB_MESSAGE = "Press E to replace lightbulb";
+    private const string FIX_LIGHT_FIXTURE = "Press E to fix light fixture and install lightbulb";
+    private const string UNSCREW_LIGHTBULB_MESSAGE = "Press E to unscrew lightbulb";
+    private const string SCREW_LIGHTBULB_MESSAGE = "Press E to screw in lightbulb";
+    public enum LightState { NotEquipped, Equipped, BrokenBulb, BrokenGlass };
+
+    [SerializeField]
+    LightState model;
+    [SerializeField]
+    public bool powered, switchState;
+    [SerializeField]
+    public Sprite NotEquippedSprite, EquippedSprite, BrokenBulbSprite, BrokenGlassSprite;
+    [SerializeField]
+    public GameObject light;
```

Comment:
Need to change light to something else

Assets/Scripts/WallLightController.cs(19,23): warning CS0108: `WallLightController.light' hides inherited member `UnityEngine.Component.light'. Use the new keyword if hiding was intended

there's already a Unity-based "light" member